.github/workflows/image.yaml

```diff
@@ -25,12 +25,13 @@ on:
       - release-*
```

Comment:
We need to remove the pull request triggers.

Reply:
done